src/main/java/kostka/moviecatalog/service/UserService.java

```diff
@@ -84,4 +85,10 @@ private String getSecuredPassword(final String password) {
         LOGGER.info("Securing password...");
         return passwordEncoder.encode(password);
     }
+
+    public boolean isUserAdultCheck(final User user) {
```

Comment:
nit: technically, this could be static as well.

Reply:
right